### DISCUSSION TITLE
Authentication with Cloudscape?

### DISCUSSION THREAD

**Discussion by Busari-Adewale** [cloudscape-design/components#2054] [General]
I found on the internet that AWS Cloudscape does not have built-in support for authentication. I understand that this functionality can be gotten by integrating AWS Amplify with Cloudscape projects. Wouldn't it be better if AWS Cloudscape has its own built-in support for authentication?

**Comment by connorlanigan** [comment]
Cloudscape itself does not interact with or rely on any backend, it only provides components to build user interfaces for the frontend. Authentication requires interaction with a backend, so it is currently out of scope for Cloudscape. That said, you can use Cloudscape components like [the Input component](https://cloudscape.design/components/input/) for building a login screen (for example, `<Input type="password ..."`)
